### PR TITLE
Improve product page

### PR DESCRIPTION
**Pull request by Hlavtox**

### CODE REVIEW

File: modules/ps_emailalerts/views/templates/hook/product.tpl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2015 PrestaShop SA
+* @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+* International Registered Trademark & Property of PrestaShop SA
+*}
+
```
Comment (kpodemski) [inline modules/ps_emailalerts/views/templates/hook/product.tpl:25]: Take a look at the new version of this file:
https://github.com/PrestaShop/ps_emailalerts/blob/dev/views/templates/hook/product.tpl

File: templates/catalog/product.tpl [hunk 2]
```diff
@@ -116,6 +117,7 @@
 
   {* SECOND PART - REASSURANCE, TABS *}
   <div class="row">
```
Comment (mparvazi) [inline templates/catalog/product.tpl:119]: Just need to add some space before row.

```suggestion
  <div class="row mt-5">
```

Comment (mparvazi) [reply templates/catalog/product.tpl:119]: Checked again, it's doesn't need any changes!
It works fine :100: 
Very nice :clap:

**Comment by mparvazi** [comment]
@Hlavtox @NeOMakinG 

We can use vertical product-thumbnails for large screen, and horizontal for mobile.
Layout is full-width, so we have enough space for vertical thumbnails in large screen.

Wdyt?

File: templates/catalog/product.tpl (hunk 2 above)
Comment (SharakPL) [reply templates/catalog/product.tpl:119]: I meant: instead of `mt-5` :)

Comment (mparvazi) [reply templates/catalog/product.tpl:119]: Try to to build and test this PR, there is no problem for spacing.
I got wrong because forget to clear cache!

**Comment by Hlavtox** [comment]
@mparvazi Well, we tested it with Valentin. It looks nice with 4 images, but when I added like 15 images like I and other clients have on many products, it was a kilometer long. :-D :-(

Maybe with some image slider it could be vertical, but if we make it horizontal, we can do it simple and just show all the thumbnails. And I think it's more UX than to hide the other thumbnails, at least the people know that the photos are there. You would not believe how many customers just don't see, don't read...

**Comment by mparvazi** [comment]
So the only problem is that, in xlarge screen may you can not see all part of the cover image in viewport.
```
    <div class="product__left col-lg-6 col-xl-7">
      {block name='product_cover_thumbnails'}
      {/block}
    </div>
```
I mean you can not see the bottom part of image, and when you try to scroll down, then you lost the top part of image, because the position of header is fixed.

I don't know are you planning to add zoom button for modal image view or not, because it somehow fix the problem.

**Comment by NeOMakinG** [comment]
@Hlavtox we need to finish this!

**Comment by NeOMakinG** [comment]
Let's go ahead on this, we will probably get back in when we will review it with the product team, if you see some improvements, don't hesitate to contribute or open an issue ❤️

File: templates/catalog/_partials/product-cover-thumbnails.tpl [hunk 3]
```diff
@@ -67,14 +67,6 @@
     </div>
   </div>
 
-
```
Comment (Hlavtox) [inline templates/catalog/_partials/product-cover-thumbnails.tpl]: @NeOMakinG 🤣🤣🤣

Comment (Hlavtox) [reply templates/catalog/_partials/product-cover-thumbnails.tpl]: "Will you remove it after we finish?" "Of course dude" 😁

Comment (NeOMakinG) [reply templates/catalog/_partials/product-cover-thumbnails.tpl]: @Hlavtox thought the same while removing it, I was like "I'm not gonna let him do this anymore."

Comment (Hlavtox) [reply templates/catalog/_partials/product-cover-thumbnails.tpl]: 🤣🤣

File: modules/ps_emailalerts/views/templates/hook/product.tpl [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{*
+* 2007-2015 PrestaShop
```
Comment (matks) [inline modules/ps_emailalerts/views/templates/hook/product.tpl:2]: 2015 👀

**Pull request merged**